sdl_android/src/main/java/com/smartdevicelink/api/screen/SoftButtonManager.java

```diff
@@ -546,4 +561,8 @@ private boolean allCurrentStateImagesAreUploaded() {
         }
         return softButtons;
     }
+
+    interface UpdateListener{
+        void onUpdate(CompletionListener completionListener);
```

Comment:
Why does this need to pass a `CompletionListener`?

Reply:
Actually, it doesn't need that param if it will be used only in the SoftButtonObjec. I will remove it.